apex/contrib/csrc/groupbn/batch_norm_add_relu.cu

```diff
@@ -167,21 +169,23 @@ at::Tensor nhwc_bn_addrelu_fwd_eval(
   // Create wrapper
   NhwcBatchNormAddRelu *bn = new NhwcBatchNormAddRelu();
 
-  bn->setInputDescriptor(CUDNN_TENSOR_NHWC, CUDNN_DATA_HALF, N, C, H, W, bn_group);
-  bn->setOutputDescriptor(CUDNN_TENSOR_NHWC, CUDNN_DATA_HALF, N, C, H, W);
+  bn->setInputDescriptor(DNN_TENSOR_FORMAT, DNN_DATA_HALF, N, C, H, W, bn_group);
+  bn->setOutputDescriptor(DNN_TENSOR_FORMAT, DNN_DATA_HALF, N, C, H, W);
 
   bn->setConstants(momentum, epsilon);
 
   // set pointers within the wrapper
-  bn->setInputOutputPointers(x.DATA_PTR<at::Half>(),
+  bn->setInputOutputPointers(x.contiguous().DATA_PTR<at::Half>(),
                              nullptr,
                              y.DATA_PTR<at::Half>(),
```

Comment:
@sarunyap Same question as previous about `y` not needing to be contiguous?

Reply:
Same reasoning:

Tensor x is the "input" tensor that is given by a user which can be non-contiguous.  Tensor y is created inside the function which is always contiguous.  So, tensor y does not need to be made contiguous before being used.